big_tests/tests/inbox_helper.erl

```diff
@@ -232,6 +239,15 @@ get_inbox_stanza(GetParams) ->
                     children = [make_inbox_form(GetParams)]},
   GetIQ#xmlel{children = [QueryTag]}.
 
+-spec get_inbox_stanza(GetParams :: inbox_query_params(), Verify :: boolean()) -> exml:element().
+get_inbox_stanza(GetParams, Verify) ->
```

Comment:
It's more of a `make_inbox_stanza` (or a thesaurus-driven equivalent) than a `get`.